### PR TITLE
Remove WebIDL from explainer.md since they are now in the spec

### PR DESCRIPTION
**Pull request by youennf**
Fixes https://github.com/w3c/webrtc-encoded-transform/issues/83

### CODE REVIEW

File: explainer.md [hunk 1]
```diff
@@ -173,97 +173,6 @@ pc.ontrack = e => {
 }
 </pre>
 
-## API
```
Comment (alvestrand) [inline explainer.md:176]: Should you put in "The detailed API proposed here is in <link>"?

Comment (youennf) [reply explainer.md:176]: I am not quite sure what your suggestion is here.
Is it to add a link to the spec?
I guess this could be done in a follow-up

Comment (alvestrand) [reply explainer.md:176]: Yes, add a link to the spec.

**Review by alvestrand — APPROVED** [review]

**Comment by youennf** [comment]
Let's do a follow-up to add the spec link to explainer.

**Pull request merged**